src/pages/Home.js

```diff
@@ -141,7 +154,19 @@ class Home extends Component {
               ])
             ])
           ]),
-          div({ className: 'row', style: { background: '#eff0f2', margin: '50px 0', padding: '60px 0 72px 0' } }, [
+          div({ className: 'row', style: { background: '#eff0f2', margin: '50px 0', padding: '48px 0 60px 0' } }, [
+            div({ className: 'row', style: { background: '#eff0f2', margin: '0 0 48px 0'} }, [
+              div({ className:'col-lg-6 col-md-6', style: { display: 'flex', justifyContent: 'center' }}, [
+                button({ className: 'btn-primary', style: buttonStyleBig }, [
+                  a({href: 'https://databiosphere.github.io/duos-ui/DatasetRegistrationInfo', style: {color: '#fff'}}, ['Register a dataset in DUOS'])
+                ])
+              ]),
+              div({ className:'col-lg-6 col-md-6', style: { display: 'flex', justifyContent: 'center' }}, [
+                button({ className: 'btn-primary', style: buttonStyleBig }, [
+                  a({href: '/dataset_catalog', style: {color: '#fff'}}, ['Submit a Data Access Request'])
```

Comment:
Might be more of a question for Jonathan, but should we show this button if a user is not logged in?

Reply:
That's a good point, this button won't work for someone not logged in, so I'll check with him.